tools/label_to_csv.py

```
@@ -1,215 +1,350 @@
+        if not split_path.is_dir():
             continue
-            # Process the files
-
-        for class_type_dir in os.listdir(dir_name):
-
-            # Check whether is dir
-            if not os.path.isdir(dir_name):
+        try:
+            class_paths = sorted(split_path.iterdir(), key=lambda path: path.name)
+        except OSError as exc:
+            raise ExportError("cannot read split directory {}: {}".format(split_path, exc))
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)